blog/2022-07-20-centralized-kubectl-access/index.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: centralized-kubectl-access
+title: "Configuring Centralized Kubectl Access To Clusters With Paralus"
+authors: [atul]
+tags: [tutorial, paralus]
+---
+
+Kubectl is one of the most widely used tools to work with Kubernetes. The command line tool allows you to deploy applications, inspect and manage resources. It basically authenticates with the control plane for your cluster and makes API calls to the Kubernetes API. In short if you are working with Kubernetes - you will use kubectl the most.
+
+In most modern day scenarios, there are multiple users who are accessing various clusters. This makes it all more important to ensure that every user or group has access to only those resources that they are allowed to. Few ways to achieve this is using namespaces and role based access control. While these are good, most enterprise grade application deployments require something more robust.
+
+<img src="/img/diagram.png" alt="Configuring Centralized Kubectl Access To Clusters With Paralus" height="70%" width="70%"/>
+
+That’s where [Paralus](https://github.com/paralus) comes in. It allows you to configure centralized kubectl access to multiple clusters all from a single dashboard. It allows you to create groups, assign projects and users and provide access. In this blog post, we’ll show you how to import different clusters to Paralus and configure access to them. All of this with zero trust principles built in.
+
+## The Use Case
+
+The goal of this blog post is to help understand how you can import multiple clusters and manage access to them from a single dashboard. In this sample use case we will have three clusters deployed in different environments. Paralus will be deployed to a cluster on Azure, two other clusters, one hosted on onprem, one localhost kind cluster and the other on Google Cloud. We’ll import these clusters into Paralus and configure centralized kubectl access to them.
+
+### Setting Up Paralus
+
+The first and foremost step here is to install Paralus. Setting up Paralus is easy with the provided helm charts. Based on the environment you are setting up Paralus on, you can follow the Paralus [installation documentation](https://www.paralus.io/docs/installation).
+
+We installed Paralus on AKS, so we created a new cluster and installed paralus using the following commands
+
+`helm repo add paralus https://paralus.github.io/helm-charts`
```

Comment:
Can we add this inside \``` \<command> ``` block? With this, it allows reader to copy command using copy button.